local/bin/py/build/configurations/pull_config_preview.yaml

```
@@ -60,7 +60,7 @@
     contents:
 
     - action: integrations
-      branch: prod
+      branch: alai97/merge-github-and-github-apps-content
```

Commit this when the Dogweb PR is merged!

```suggestion
      branch: prod
```